src/styles/_scripts.scss

```diff
@@ -0,0 +1,17 @@
+/* ================ 3rd Party Script styling overrides ================ */
+
+/**
+ * Needed to make sure cookie banner doesn't have white text on white background
+ * (Cards for the website are configured to have white background, but body text is also configured to be white).
+ * stylelint-disable-next-line selector-max-id
+ */
+#CybotCookiebotDialog {
+    color: $dark-9;
+    opacity: 0.95;
+    z-index: 30 !important;
```

Comment:
Nothing to change here, but fun facts; I found this IAB standard with a table for [managing z-index](https://www.oreilly.com/library/view/developing-web-components/9781491905685/ch04.html#idp7656464:~:text=index%20management%20increases.-,Managing%20z%2Dindexes,-A%20common%20approach). We probably don't have to get this out of hand lol but maybe we could come up with our own standard using tens or hundreds and document it somewhere? I know [Bootstrap starts at 1000](https://getbootstrap.com/docs/5.0/layout/z-index/), and there's [a max documented by DO](https://www.digitalocean.com/community/tutorials/css-z-index#:~:text=The%20maximum%20range%20is%20%C2%B12147483647.). 🤓

Reply:
Ah I like that. I changed the z-index values to fall in line with Bootstraps standards. Maybe after our Tailwind conversion we should mention Tailwind in the README as part of our stack & mention the CSS method along with the z-index standard we're implementing?

Reply:
Sounds like a plan! After Tailwind is in, I'd love to follow [their implementation for z index](https://tailwindcss.com/docs/z-index). Seems much more simplistic with the numbers in tens 🤓

Reply:
I was using Bootstrap's since they're indicating values for different kinds of content (popups, tooltips, etc). But I agree numbers in the tens are simpler & we can use our own dictionary off Tailwinds increments 🔢 😄 !